[Workspace] Add documentation for workspace and ACL (#8643)
opensearch-project/documentation-website: _dashboards/workspace/manage-workspace.md

---
layout: default
title: Manage workspaces
parent: Workspace
nav_order: 2
---

> # Manage workspaces

Reviewer: ```suggestion
# Manage workspaces
Introduced 2.18
{: .label .label-purple }
```

    - **With security plugins installed and `savedObjects.permission.enabled: true` in the `config/opensearch_dashboards.yml` file:** Only the OpenSearch Dashboards admin can delete the workspace.
2. **Set as default workspace:** Sets the current workspace as the default login destination.
3. **Workspace overview:** Opens the workspace overview page in a new tab.

## Adding assets to the workspace

Access the **Sample data** in the navigation menu on the left. Select the appropriate dataset to install it in your cluster and OpenSearch Dashboards.

## Copying assets between workspaces

Data sources and configuration copying are not supported.
{: .warning}

The assets page provides the following methods for copying assets across workspaces:

1. **Copy all assets to...:** Copies all assets in the table.
2. **Copy to...:** Moves selected assets from the table.
3. **Copy to...:** Copies a single asset from the table.

After selecting a copy option, choose the target workspace from the dropdown menu. The **Copy related assets** checkbox allows you to transfer associated assets.

> 

Reviewer: ```suggestion
```

---
layout: default
title: Manage workspaces
parent: Workspace
nav_order: 2
---

# Manage workspaces
Introduced 2.18
{: .label .label-purple }

You can access and modify the workspace details including name, description, use case, and icon color on the details page. 

> 

Reviewer: **Workspace details**

> 

Reviewer: I'll check the "page" references to make sure we're using the proper name.

To access and modify your workspace details, follow these steps: 

1. Open OpenSearch Dashboards and navigate to **Workspaces**.
2. Choose the **Edit** button to make changes, and then choose the **Save** button to confirm changes or the **Discard changes** button to cancel modifications.

## Workspace update permissions

The following permissions apply when changing workspaces:

> 1. **Without security plugins:** All users can edit and update the workspace.

Reviewer: ```suggestion
1. **Without the Security plugin:** All users can edit and update the workspace.
```

2. **With security plugins installed and `savedObjects.permission.enabled: false` in the `config/opensearch_dashboards.yml` file:** All users can edit and update workspaces.

> 3. **With security plugins and `savedObjects.permission.enabled: true` in the `config/opensearch_dashboards.yml`:** Only the [workspace owner]({{site.url}}{{site.baseurl}}/dashboards/workspace/workspace-acl/#define-collaborators-for-your-workspaces) and the [OpenSearch Dashboards admin]({{site.url}}{{site.baseurl}}/dashboards/workspace/workspace-acl/#config-dashboard-admin) can edit and update workspaces.

Reviewer: ```suggestion
3. **With the Security plugin and `savedObjects.permission.enabled: true` in the `config/opensearch_dashboards.yml`:** Only the [workspace owner]({{site.url}}{{site.baseurl}}/dashboards/workspace/workspace-acl/#define-collaborators-for-your-workspaces) and the [OpenSearch Dashboards admin]({{site.url}}{{site.baseurl}}/dashboards/workspace/workspace-acl/#config-dashboard-admin) can edit and update workspaces.
```

## Workspace update restrictions 

When updating workspace use cases, the following rules apply.

Original use case | Target use case |
:---: | :---:
Analytics  | Cannot be changed to any other use case
Search  | Analytics
Security analytics  | Analytics
Observability  | Analytics

> Essentials  |    Analytics Search<br> Security analytics<br> Observability

Reviewer: ```suggestion
Essentials  |    Analytics Search<br> Security Analytics<br> Observability
```

## Workspace control panel

The workspace details page features the following buttons in the upper-right corner:

1. **Delete** (trash icon)

>     - **Without security plugins uninstalled:** All users can delete the workspace.

Reviewer: ```suggestion
    - **Without the Security plugin installed:** All users can delete the workspace.
```

>     - **With security plugins installed and `savedObjects.permission.enabled: false` in the `config/opensearch_dashboards.yml` file:** All users can delete the workspace.

Reviewer: ```suggestion
    - **With the Security plugins installed and `savedObjects.permission.enabled: false` in the `config/opensearch_dashboards.yml` file:** All users can delete the workspace.
```

    - **With security plugins installed and `savedObjects.permission.enabled: true` in the `config/opensearch_dashboards.yml` file:** Only the OpenSearch Dashboards admin can delete the workspace.
2. **Set as default workspace:** Sets the current workspace as the default login destination.
3. **Workspace overview:** Opens the workspace overview page in a new tab.

## Adding assets to the workspace

Access the **Sample data** in the navigation menu on the left. Select the appropriate dataset to install it in your cluster and OpenSearch Dashboards.

## Copying assets between workspaces

> Data sources and config copying are not supported.

Reviewer: ```suggestion
Data sources and configuration copying are not supported.
```

{: .warning}

The assets page offers the following methods for copying assets across workspaces:

1. **Copy all assets to...:** Copies all assets in the table.
2. **Copy to...:** Moves selected assets from the table.
3. **Copy to...:** Copies a single asset from the table.

After selection a copy option, choose the target workspace from the dropdown menu. The **Copy related assets** checkbox allows for transfer of associated assets.

Copying permissions vary based on security plugin status and configuration.

> {: .note}

Reviewer: deleted

> Copying permissions vary based on Security plugin status and configuration.

Reviewer: ```suggestion
Your copy permissions are based on the Security plugin status and configuration.
```

> Your copy permissions are based on the Security plugin status and configuration.

Reviewer: ```suggestion
```
